src/uri/file.lisp

```diff
@@ -12,7 +12,15 @@
            :uri-file-pathname))
 (in-package :quri.uri.file)
 
-(defstruct (uri-file (:include uri (scheme "file") (port nil))))
+(defstruct (uri-file (:include uri (scheme "file") (port nil))
+                     (:constructor %make-uri-file)))
+
+(defun make-uri-file (&rest initargs)
```

Comment:
I'm thinking that this could be `&rest initargs &key path` and then processing the path in place in the `apply #'%make-uri-file`.

Reply:
Done, what do you think?

Reply:
Yes, that's good :)